ci/jenkins/README.md

```diff
@@ -105,14 +113,36 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-conformance --registry ${DOCKER_REGISTRY}
 ```
 
-* windows network policy: community tests on Windows cluster with Docker runtime, focusing on "Feature:NetworkPolicy".
+* windows containerd conformance: community tests on Windows cluster with Containerd runtime
+  and OVS running inside container, focusing on "Conformance|sig-windows" and
+  "sig-network", and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage",
+  "sig-auth", "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from",
+  "[sig-network] Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints
```

Comment:
@wenyingd The endpoints latency case still failed in proxy-all mode.

Reply:
> @wenyingd The endpoints latency case still failed in proxy-all mode.

Could you create an issue to track it? Maybe we need more investigation as no kube-proxy is used on Windows now.

Reply:
Created PR https://github.com/antrea-io/antrea/pull/5773 to fix it.

Reply:
@XinShuYang , wrt #5773, since "should be able to change the type from|\[sig-network\] Services should be able to create a functioning NodePort service \[Conformance\]|Service endpoints latency should not be very high|should be able to create a functioning NodePort service for Windows " is removed from test script, @jainpulkit22 ,will need to update this to readme, alongside a line shld be added wrt usage of proxy-all mode.

Reply:
@jainpulkit22 ,
as PR #5773 Re-enables few Windows conformance tests skipped due to non usage of kube-proxy mode on windows, so need to remove skip section here"should be able to change the type from|\[sig-network\] Services should be able to create a functioning NodePort service \[Conformance\]|Service endpoints latency should not be very high|should be able to create a functioning NodePort service for Windows"